### DISCUSSION TITLE
Personnalisation du formulaire - Bouton Déconnexion

### DISCUSSION THREAD

**Discussion by MpmMess** [MTESSDev/FRW#10] [Annonces]
Est-ce que l'on peut changer le libellé du bouton "Déconnexion" pour "Fermer" et que cela ferme la fenêtre?

**Comment by j0ber** [comment]
Aujourd'hui non et à première vue ce n'est pas une orientation que nous voulons prendre. 
L'idée est que tous les formulaires aient un visuel identique. 

Aucune raison pour que certains formulaires aient un bouton "Fermer" au lieu de "Déconnexion". 
La déconnexion confirme à l'utilisateur qu'il s'est déconnecté correctement (c'est rassurant pour lui) et c'est un concept largement répandu sur le web. On ne voit plus ça des sites qui ferment l'onglet courant sur un clic d'un bouton.

Aussi fermer un onglet avec du javascript n'est maintenant possible que si cet onglet a été ouvert avec un window.open (chose sur laquelle nous n'avons pas de contrôle).
